src/transformers/utils/hub.py

```diff
@@ -811,6 +822,8 @@ def push_to_hub(
                 Whether or not to create a PR with the uploaded files or directly commit.
             safe_serialization (`bool`, *optional*, defaults to `False`):
                 Whether or not to convert the model weights in safetensors format for safer serialization.
+            revision (`str`, *optional*, defaults to None):
```

Comment:
```suggestion
            revision (`str`, *optional*):
```
Nit: we don't write the default when it's None as it's implied by optional.

Reply:
Removed!